scripts/ec2cleanup.py

```diff
@@ -218,8 +220,30 @@ def delete_queues(provider_mgmt, excluded_queues, output):
         logger.exception('Exception in %r', delete_queues.__name__)
 
 
+def delete_s3_bucket(provider_mgmt, bucket_name, output):
+    bucket_list = []
+    provider_name = provider_mgmt.kwargs['name']
+    try:
+        for bucket in provider_mgmt.list_s3_bucket():
+            if bucket_name in bucket:
+                deleted_bucket = provider_mgmt.delete_s3_buckets(delete_all_objects=True,
```

Comment:
REQUIRED: your wrapanapi call returns bool. Why do you try to collect name here ?

Reply:
Right now made wrapanapi to return list of names rather objects.